src/libstore/globals.hh

```diff
@@ -275,6 +283,27 @@ public:
           multi-user settings with untrusted users.
         )"};
 
+    Setting<bool> autoAllocateUids{this, false, "auto-allocate-uids",
+        "Whether to allocate UIDs for builders automatically."};
+
+    Setting<uint32_t> startId{this,
+        #if __linux__
+        0x34000000,
+        #else
+        56930,
```

Comment:
Also, maybe change this to hex as well for symmetry?

Reply:
This one is pretty random (0xde62) so not very interesting in hex.